static/js/components/tiles/histogram_tile.tsx

```diff
@@ -179,7 +180,8 @@ function binData(
     let eventValue = 1;
     if (property) {
       // default to 0 if property can't be found in display props
-      eventValue = event.displayProps[property] || 0;
+      eventValue =
+        stripUnitFromPropertyValue(event.displayProps[property]) || 0;
```

Comment:
should check that event.displayProps[property] is not undefined?

Reply:
Added check!